### ISSUE TITLE
[TODO][QoL][Testing] Accessibility review of codestream

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/codestream#110]
We're lacking in some accessibility highlighted by

- Firefox accessibility tab in console
- Google Lighthouse
- Google PageSpeed (the other Lighthouse)
- W3C browser extension ( called WAVE)

Let's test, review all ARIA meta.

Goal is 95% or better on all tests

**Comment by Sriram2272** [comment]
Hi, I’m Dama Sriram from LPU. I’d like to work on this issue if it’s still available.
Could you please assign it to me?
